NomadicNibbler/foodtruck-fe
# Feature/service worker

## What does this PR do?
- A service worker file was added to the public folder
  - A service worker registration script was added to the index.html
  - Now when the app is loaded, a service worker will be registered and will cache all current static files 
  - When offline, the application will load the static files from the cache
- the site.webmanifest file was updated with all the proper application information to render the correct icons and desired colors

## How should it be tested?
- Pull the branch down and load the page with internet, open up the console. You should see a notification saying the service worker was registered
- Go to the application tab and click on service worker, turn the internet off, reload the page, the page should still display all current routes
- The favicon should now be displaying the orange truck icon

## What do the future iterations hold?
- We will need to set up the service worker to cache/store api data when in offline mode
- We will need to flesh out the service worker file to update the cache version

## Linked Issues
- #38
